composer.json

```diff
@@ -7,64 +7,54 @@
     "license": "BSD-3-Clause",
     "minimum-stability": "stable",
     "require": {
-        "php": ">=7.0.0",
-
-        "yiisoft/yii2": "~2.0.36",
-
-        "yiisoft/yii2-apidoc": "~2.1.6",
-        "cebe/markdown-latex": "^1.1.3@dev",
-        "cebe/markdown": "~1.0.3",
-
-        "yiisoft/yii2-authclient": "~2.2.0",
-        "yiisoft/yii2-httpclient": "~2.0.0",
-        "yiisoft/yii2-elasticsearch": "~2.1.0",
-        "yiisoft/yii2-redis": "~2.0.0",
-        "yiisoft/yii2-queue": "~2.0.0",
-        "yiisoft/yii2-swiftmailer": "~2.0.0",
-        "cebe/yii2-gravatar": "1.1",
-        "creocoder/yii2-flysystem": "~0.9.0",
-
-        "samdark/sitemap": "^2.0",
-
-        "knplabs/github-api": "^2.11",
-        "php-http/curl-client": "^1.7",
-        "guzzlehttp/psr7": "^1.5",
-
-        "scrivo/highlight.php": "~9.18",
-        "yetanotherape/diff-match-patch": "~1.0.0",
-
-        "imagine/imagine": "^0.6.3",
-
-        "2amigos/yii2-taggable-behavior": "~1.0.1",
         "2amigos/yii2-selectize-widget": "~1.0.1",
-
-        "yiisoft/yii2-bootstrap": "~2.0.6",
-        "yiisoft/yii2-jui": "~2.0.6",
+        "2amigos/yii2-taggable-behavior": "~1.0.1",
+        "abraham/twitteroauth": "~0.7.4",
+        "baibaratsky/yii2-rollbar": "^1.7",
         "bower-asset/blueimp-file-upload": "~9.22.0",
-
-        "bower-asset/jquery": "2.2.*@stable",
         "bower-asset/bootstrap-sass": "3.3.6",
-        "bower-asset/font-awesome": "^4.5",
         "bower-asset/bootstrap-social": "^4.11",
+        "bower-asset/codemirror": "~5.9.0",
+        "bower-asset/codemirror-buttons": "~1.0.0",
+        "bower-asset/font-awesome": "^4.5",
         "bower-asset/glidejs": "^2.0",
+        "bower-asset/jquery": "2.2.*@stable",
         "bower-asset/sass-rem": "^1.2",
         "bower-asset/scrollup": "^2.4",
-        "bower-asset/codemirror": "~5.9.0",
-        "bower-asset/codemirror-buttons": "~1.0.0",
+        "cebe/markdown": "~1.0.3",
+        "cebe/markdown-latex": "^1.1.3@dev",
+        "cebe/yii2-gravatar": "1.1",
         "composer/spdx-licenses": "^1.1",
-        "zendframework/zend-feed": "^2.8",
-        "abraham/twitteroauth": "~0.7.4",
-        "baibaratsky/yii2-rollbar": "^1.7",
-        "himiklab/yii2-recaptcha-widget": "^1.2",
+        "creocoder/yii2-flysystem": "~0.9.0",
+        "guzzlehttp/psr7": "^1.5",
         "helgesverre/spamprotection": "1.0.1",
-        "yiisoft/yii2-imagine": "~2.1.0"
+        "himiklab/yii2-recaptcha-widget": "^1.2",
+        "imagine/imagine": "^0.6.3",
+        "knplabs/github-api": "^2.11",
+        "php": "^7.4",
+        "php-http/curl-client": "^1.7",
+        "samdark/sitemap": "^2.0",
+        "scrivo/highlight.php": "~9.18",
+        "yetanotherape/diff-match-patch": "~1.0.0",
+        "yiisoft/yii2": "~2.0.44",
```

Comment:
The only changes were:

```json
"yiisoft/yii2": "~2.0.44",
"yiisoft/yii2-apidoc": "dev-master"
```
Reordering is applied for readability purposes.

Reply:
We can force the order: https://github.com/yiisoft/di/blob/master/composer.json#L63

Reply:
Thanks. Added. :ok_hand: